rollup.config.js

```diff
@@ -31,6 +36,7 @@ const configBase = {
   globals: { react: 'React' },
   external: ['react'].concat(Object.keys(pkg.dependencies)),
   plugins: commonPlugins,
+  sourcemap: true,
```

Comment:
Hi folks, 

I believe you have typo here. It should be `sourceMap` instead of `sourcemap` see https://github.com/maxdavidson/rollup-plugin-sourcemaps.

cc @philpl

Reply:
I'm pretty sure Rollup updated their config to use the lowercase version. If you use `sourceMap` you'll get a warning.

Reply:
No worries! :blush: